docs/source/upgrade-guide/index.md

```diff
@@ -89,6 +89,26 @@ If you want to retain the old behavior (and no use `apiExpanders` at all), you n
 config.settings.apiExpanders = [];
 ```
 
+### Cypress upgraded to 12.17.1
+
+As usual in a Volto major version, Cypress has been upgraded to latest version to date.
+There are no major changes to the way the tests are implemented and run.
+
+However, could be that the Cypress boilerplate you are using has to be updated in your projects and add-ons in case you are using `@testing-library/cypress` in your tests.
```

Comment:
```suggestion
However, it could be that your Cypress boilerplate must be updated in your projects and add-ons if you use `@testing-library/cypress` in your tests.
```